accessibility-checker-engine/help/IBMA_Focus_Tabbable.mdx

```diff
@@ -8,11 +8,11 @@ import { CodeSnippet, Tag } from "carbon-components-react";
 <Row>
 <Column colLg={16} colMd={8} colSm={4} className="toolHead">
 
-### The component does not have a tabbable element
+### The component does not have a tabable element
```

Comment:
tabbable was correct

Reply:
agree. These shoud not have been changed. correcting back